windows/src/unit-tests/package-info/.gitignore

```diff
@@ -0,0 +1,2 @@
+test/*.out.*
+Win32
```

Comment:
can we consolidate .gitignore to the root platform directory?

Reply:
I'm going to suggest that minimal localised .gitignore are less side-effect prone and help us to see that files are ignored in the folders we are working in. When it's a top-level .gitignore, then we tend not to see it...

Reply:
(I have, in a separate PR, consolidated the Win32 folder to a parent level folder because there are two projects that use it and it really is a common ignore, unlike the test/*.out.* filespec)